content/en/security/application_security/threats/setup_and_configure.md

```diff
@@ -231,6 +231,46 @@ Support for query strings is not available for Flask.
 
 {{< /programming-lang >}}
 
+{{< programming-lang lang="serverless" >}}
+
+<div class="alert alert-info">ASM support for AWS Lambda is in beta. Threat detection is done by using the Lambda extension.<br><br> Don’t see your desired technology listed here? Datadog is continually adding additional support. Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send us details</a>.</div>
+
+
+### Supported cloud environments
+
+- AWS Lambda (beta)
+
+### Supported languages and their requirements
+
```

Comment:
Add a new section for version dependencies. 

Extension version: `38`
Serverless plugin version: `v5.19.0`

Reply:
Extension version is `39` in the end.